kubernetes/internal/domain-custom-resource-template.yaml

```diff
@@ -35,6 +35,13 @@ spec:
   asName: "%ADMIN_SERVER_NAME%"
   # The Admin Server's ListenPort
   asPort: %ADMIN_PORT%
+  # Whether to include the server out file into the pod's stdout
+  includeServerOutInPodLog: "%INCLUDE_SERVER_OUT_IN_POD_LOG%"
+  # The in-pod name of the directory to store the domain, node manager, server logs, and server .out
+  # files in.
+  # If not specified or empty, then these files will be stored in their normal locations under the
+  # domain home.
+  logHome: %LOG_HOME%
```

Comment:
Comment is incorrect, the operator implementation diff for the Operator pod below seems to be putting the log in /shared/logs/[domain-uid] if blank or unspecified.